frontend/tasks/content.js

```diff
@@ -161,7 +161,7 @@ function buildExperimentsData() {
 }
 
 .experiment-icon-${experiment.slug} {
-  background-image: url(${experiment.thumbnail});
+  background-image: url(../..${experiment.thumbnail});
```

Comment:
Kind of a hack, [but a relative path works much better with the Storybook deployment script](http://storybook.testpilot.dev.mozaws.net/6a7fca5b3e30612dd561adf631ae28641450ef8e/?knob-Has%20Addon=false&knob-Enabled=false&knob-Experiment=%7B%22title%22%3A%22Sample%20experiment%22%2C%22description%22%3A%22This%20is%20an%20example%20experiment%22%2C%22subtitle%22%3A%22%22%2C%22slug%22%3A%22snooze-tabs%22%2C%22survey_url%22%3A%22https%3A%2F%2Fexample.com%22%2C%22installation_count%22%3A%2286753%22%2C%22created%22%3A%222010-06-21T12%3A12%3A12Z%22%2C%22modified%22%3A%222010-06-21T12%3A12%3A12Z%22%7D&selectedKind=ExperimentRowCard&selectedStory=has%20addon%20%26%20enabled&full=0&down=1&left=1&panelRight=0&downPanel=storybook%2Factions%2Factions-panel).